crates/bindings-csharp/Runtime/Internal/Module.cs

```diff
@@ -252,15 +252,33 @@ private static byte[] Consume(this BytesSource source)
                 // but a module should be prepared for it.
                 case 0:
                     break;
+                case (short)(ushort)FFI.Errno.NO_SUCH_BYTES:
```

Comment:
Same questions as in the other PR about just changing Errno to `short` and whether `(short)FFI.Errno.` would suffice here.